tests/fmt.rs

```diff
@@ -0,0 +1,23 @@
+#[test]
+fn fmt() {
+    if option_env!("RUSTC_TEST_SUITE").is_some() {
+        return;
+    }
+
+    let root_dir = std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    let dev_dir = root_dir.join("clippy_dev");
+    let output = std::process::Command::new("cargo")
+        .current_dir(dev_dir)
+        .args(&["run", "--", "fmt", "--check"])
```

Comment:
Should we specify here to run nightly `rustfmt`?

```suggestion
        .args(&["+nightly", "run", "--", "fmt", "--check"])
```

Reply:
Never mind, this is already specified in `cargo_fmt(..)` and `rustfmt(..)`.